pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -158,6 +156,9 @@ class Search extends Component<IAppState, ISearchState> {
                   </Level>
 
                 )}
+                
+                {console.log("res ui: ",this.state.results)}
```

Comment:
Unless this is still necessary for some reason, can you delete this line?

Reply:
This does not seem necessary now, I have removed it.